tests/unit/storages/conftest.py

```diff
@@ -1,17 +1,40 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
 import pytest
 
 from crawlee import service_locator
-from crawlee.storage_clients import FileSystemStorageClient, MemoryStorageClient, SqlStorageClient, StorageClient
+from crawlee.storage_clients import (
+    FileSystemStorageClient,
+    MemoryStorageClient,
+    RedisStorageClient,
+    SqlStorageClient,
+    StorageClient,
+)
+
+if TYPE_CHECKING:
+    from fakeredis import FakeAsyncRedis
 
 
-@pytest.fixture(params=['memory', 'file_system', 'sql'])
-def storage_client(request: pytest.FixtureRequest) -> StorageClient:
+@pytest.fixture(params=['memory', 'file_system', 'sql', 'redis_default', 'redis_bloom'])
```

Comment:
I'm not sure if it makes sense to have two Redis storage clients here, as that would result in many more test runs covering the same functionality, just due to a different type of deduplication in Redis RQ. Maybe we could test Redis RQ with both the default and bloom deduplication in a more targeted way instead.